src/validate.ts

```diff
@@ -9,7 +9,16 @@ const SYNC_RESPONSE_SCHEMA = require('../intents/sync.response.schema.json');
 const QUERY_RESPONSE_SCHEMA = require('../intents/query.response.schema.json');
 const EXECUTE_RESPONSE_SCHEMA = require('../intents/execute.response.schema.json');
 const DISCONNECT_RESPONSE_SCHEMA = require('../intents/disconnect.response.schema.json');
+const ON_OFF_STATES_SCHEMA = require('../traits/onoff.states.schema.json');
+const ON_OFF_ATTRIBUTES_SCHEMA = require('../traits/onoff.attributes.schema.json');
 
+const COMMAND_STATES_EXPECT = {
+  'action.devices.commands.OnOff': ON_OFF_STATES_SCHEMA,
+};
+
+const TRAIT_ATTRIBUTES_EXPECT = {
+  'action.devices.traits.OnOff': ON_OFF_ATTRIBUTES_SCHEMA,
```

Comment:
Do you need access to the other schema files so you can fully validate your examples?